static/app/views/alerts/rules/issue/previewChart.tsx

```diff
@@ -0,0 +1,105 @@
+import styled from '@emotion/styled';
+
+import {AreaChart} from 'sentry/components/charts/areaChart';
+import ChartZoom from 'sentry/components/charts/chartZoom';
+import {HeaderTitleLegend, SectionHeading} from 'sentry/components/charts/styles';
+import {Panel, PanelBody, PanelFooter} from 'sentry/components/panels';
+import Placeholder from 'sentry/components/placeholder';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {ProjectAlertRuleStats} from 'sentry/types/alerts';
+import getDynamicText from 'sentry/utils/getDynamicText';
+
+type Props = {
+  ruleFireHistory: ProjectAlertRuleStats[];
+};
+
+const PreviewChart = (props: Props) => {
+  const renderChart = ruleFireHistory => {
+    const series = {
```

Comment:
adding a type here prevents typescript from creating a new type and comparing it
```suggestion
    const series: AreaChartSeries = {
```